### ISSUE TITLE
Axe-core contrast error in header menu

### ISSUE THREAD

**Issue by dmundra** [uswds/uswds#4410]
### Describe the bug
We are using USWDS Jekyll theme with a dropdown header. This example https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/components/preview/header--basic.html from https://designsystem.digital.gov/components/header/ matches our site.

When running Axe-core accessibility testing against the header it says that the menu item does not have sufficient contrast (https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome). I believe that error is being caused by the transparent down arrow.

### Steps to reproduce the bug
1. Download the Axe-core browser extension.
2. Go to https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/components/preview/header--basic.html
3. Run the Axe-core test for the entire page.

### Expected behavior
I think maybe the contrast on the menu icon could be increase or some type of similar solution. Unless this is a false negative.

### Screenshots
<img width="1242" alt="Screen Shot 2021-11-29 at 8 04 08 PM" src="https://user-images.githubusercontent.com/7096681/143984527-d3a6e4f3-4beb-4355-b83f-4440291cfe5c.png">

### System setup
- **USWDS version:** 2.12.2
- **Device:** Macbook Pro
- **Operating system:** Mac OS X
- **Browser and version:** Chrome

### Additional context
If you remove the menu icon the contrast error does not appear.

**Comment by aduth** [comment]
This seems like a false positive. As noted in the screenshot, the issue requires review because the extension can't determine contrast due to a combination of a background gradient and background image. The icon itself is black on a white background, which is quite a lot of contrast!

https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FFFFFF

**Comment by dmundra** [comment]
I agree with that @aduth. I have noticed the false positive with contrast checks in axe-core.
